packages/core/src/client.ts

```diff
@@ -896,7 +903,7 @@ export class MedplumClient extends TypedEventTarget<MedplumClientEventMap> {
 
     if (this.storage.getInitPromise === undefined) {
       if (!options?.accessToken) {
-        this.attemptResumeActiveLogin().catch(console.error);
+        this.attemptResumeActiveLogin().catch(this.logger.error);
```

Comment:
This is slightly different, as errors in console will no longer show up as console errors but just regular log statements. If we can find away to preserve the mapping to `console.{log, warn, error}` that would be ideal